aws/resource_aws_workspaces_workspace_test.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 func init() {
-	resource.AddTestSweepers("aws_workspace", &resource.Sweeper{
-		Name: "aws_workspace",
+	resource.AddTestSweepers("aws_workspaces_workspace", &resource.Sweeper{
```

Comment:
I think the SDK v2 upgrade may require this fix, it would be good to separate it so we can merge it in. 👍

Reply:
Created #14165